.github/workflows/pull-request-account-compression.yml

```
@@ -0,0 +1,89 @@
+name: Account Compression Pull Request
+
+on:
+  pull_request:
+    paths:
+      - "account-compression/**"
+      - "ci/*-version.sh"
```

To be safer, what do you think?
```suggestion
      - "account-compression/**"
      - "libraries/concurrent-merkle-tree/**"
      - "ci/*-version.sh"
```